docs/readthedocs/source/paredit.md

```diff
@@ -122,6 +122,8 @@ Care has been put in to making the default keybindings somewhat logical, easy to
 
 Note: You can choose to disable all default key bindings by configuring `calva.paredit.defaultKeyMap` to `none`. (Then you probably also want to register your own shortcuts for the commands you often use.)
 
+You can toggle Paredit on and off using the `paredit.toggleenabled` command. This allows you to switch modally between Paredit keybindings and standard textual keybindings.
```

Comment:
Just a note here, the current docs are now in the calva-docs repo, and should be updated there. We need to see about removing these old docs so as not to cause confusion.

Reply:
Noted, I will remove this change from the PR. I can submit another PR if you like which deletes all the docs and updates the PR template to point people towards calva-docs instead if you'd like?